readme.txt

```diff
@@ -101,6 +101,9 @@ Please note that our support for the checkout block is still experimental and th
 
 == Changelog ==
 
+= 2.1.0 - 2021-xx-xx =
+* Add - Add UI styling and notices for instant deposits.
```

Comment:
We'll be launching the whole Instant Deposits feature on the same release, so the changelog should reflect that, not only the contents of this PR.

Reply:
Updated 👍